Message/tooltip changes

Bertie690 opened the pull request:
Fixed some message/tooltip bugs that IDK how nobody noticed for this long

Bertie690 committed:
Message/tooltip changes

Fixed some message/tooltip bugs that IDK how nobody noticed for this long

sirgwain reviewed (changes requested):
The package json, lock files, and go.mod shouldn’t have changed for these fixes. It was probably something that happened when you setup for local dev. Please revert.

Bertie690 commented:
> The package json, lock files, and go.mod shouldn’t have changed for these fixes. It was probably something that happened when you setup for local dev. Please revert

Sure. 
In that case, would it make sense to add em to the `.gitignore` to prevent github from detecting changes in them?

Bertie690 commented:
Also for the record I'm still sketchy on reverting the changes to `go.mod` & `go.sum` since the github page for `mattn/go-sqlite3` explicitly stated that v2.0 was a numbering mistake and version 1.14 was in fact the latest version. 
Given that these two files more or less appear to be a dependency list of sorts (keyword "more or less"), I dunno if having an outdated file as "required" really makes sense given that go automatically downgraded the driver the moment I installed it...

Bertie690 committed:
Revert file changes

Partial revert of commit c4d4dea29e184dcbb478a37332c4a610d8432895 to undo changes to stuff
Also made minor comment typo fixes

sirgwain commented on frontend/src/lib/components/game/tooltips/PopulationTooltip.svelte:
```diff
@@ -24,34 +24,38 @@
 				<span class="font-semibold">{planet.spec.population.toLocaleString()}</span> ({(
 					planet.spec.populationDensity * 100
-				).toFixed()}% of capacity).
+				).toFixed()}% of capacity).<br />
```
instead of using <br>, please wrap anything that needs to be a separate paragraph in a <p>content</p>

Bertie690 replied:
Ok I'll get on to that once I get back home and set up the local instance again. 
Will have to see how it looks first though.

Bertie690 committed:
Replaced line breaks with paragraph tags

Bertie690 committed:
Update ARM description

Minor change to make ARM mention how exactly their fancy toys are better than normal remote miners (namely their efficiency, cheapness and lower weight).
